integration-tests/opentelemetry-logging/src/test/resources/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+
+services:
```

Comment:
probably no need for a docker composer on this IT test given that the test is using the  `InMemoryLogRecordExporter`.

Reply:
we already have that test in [opentelemetry-vertx-exporter](https://github.com/quarkusio/quarkus/blob/b6f54993c43f955f2b20855b9c4d694af7c210f2/integration-tests/opentelemetry-vertx-exporter/src/test/java/io/quarkus/it/opentelemetry/vertx/exporter)

Reply:
Dones, I added tests there and remove the files